bazel_versions.bzl

```diff
@@ -3,8 +3,7 @@
 CURRENT_BAZEL_VERSION = "//:.bazelversion"
 
 OTHER_BAZEL_VERSIONS = [
-    "6.0.0-pre.20221020.1",
```

Comment:
does this mean no one can use rules_integration_testing to test against bazel 6? that seems like a regression if so

Reply:
Created #122